datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -2925,4 +2927,10 @@ private void configureRestrictedResolvers(final RuntimeWiring.Builder builder) {
                         siblingGraphService, restrictedService, this.authorizationConfiguration))
                 .dataFetcher("relationships", new EntityRelationshipsResultResolver(graphClient)));
   }
+
+  private void configureActorResolvers(final RuntimeWiring.Builder builder) {
```

Comment:
small one: but maybe let's rename this to `configureRoleResolvers` and rename the current `configureRoleResolvers` to be `configureDataHubRoleResolvers`